src/pages/eth2/scams.js

```diff
@@ -0,0 +1,207 @@
+      date: "11/05/2020",
+      platform: "discord",
+    },
+  ]
+  return (
+    <StyledPage>
+      <PageMetadata
+        title="Eth2 staking scams"
+        description="A place to check and report known Eth2 staking scams."
+      />
+      <Header>
+        <div>
+          <Title>Eth2 staking scams</Title>
+          <Subtitle>
+            Unfortunately scams are inevitable. Here, you can check known scams
+            or report any for the wider staking community.
+          </Subtitle>
+        </div>
+        <Modal isOpen={isModalOpen} setIsOpen={setModalOpen}>
+          <ModalTitle>Report scam</ModalTitle>
+          <p>
+            If you've experienced or seen a scam, please report it to help
+            protect the staking community. You'll need to use GitHub or let us
+            know via{" "}
+            <Link to="https://discord.gg/CetY6Y4">our Discord channel</Link>.
+          </p>
+          <ModalBody>
+            <ModalOption>
+              <p>
+                <b>New to GitHub?</b>
+                <br />
+                Raise an issue – just fill in the requested information.
+              </p>
+              <GithubButton
+                isSecondary
+                to="https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=Type%3A+Feature&template=suggest_tutorial.md&title="
+              >
+                <GithubIcon name="github" /> <span>Raise issue</span>
+              </GithubButton>
+            </ModalOption>
+            <ModalOption>
+              <p>
+                <b>Create a pull request</b>
+                <br />
+                Add the information and update the site yourself.
+              </p>
+              <GithubButton
+                isSecondary
+                to="https://github.com/ethereum/ethereum-org-website/new/dev/src/pages/eth2/scams/"
+              >
+                <GithubIcon name="github" /> <span>Create pull request</span>
+              </GithubButton>
+            </ModalOption>
+          </ModalBody>
+        </Modal>
+        <ButtonSecondary onClick={() => setModalOpen(true)}>
+          Report scam
+        </ButtonSecondary>
+      </Header>
+      <ScamCard content={scams} />
+      {/* <Image fixed={data.discordscam.childImageSharp.fixed} /> */}
+    </StyledPage>
+  )
+}
+
+export default KnownScamsPage
+
+/* export const scamImage = graphql`
+  fragment scamImage on File {
+    childImageSharp {
+      fixed(width: 600) {
+        ...GatsbyImageSharpFixed
+      }
+    }
+  }
+`
+export const query = graphql`
+  query {
+    discordscam: file(relativePath: { eq: "eth2-scams/discord-scam.png" }) {
+      ...scamImage
+    }
+  }
+` */
+
+export const query = graphql`
+  query {
+    discordscam: file(relativePath: { eq: "eth2-scams/discord-scam.png" }) {
+      childImageSharp {
+        fixed(width: 400) {
+          ...GatsbyImageSharpFixed
+        }
+      }
+    }
+  }
+`
```

Comment:
![image](https://user-images.githubusercontent.com/54227730/98599378-b909e300-2290-11eb-8e43-68df3d2f35ac.png)

Those changes allow the image to render on my end.

Reply:
(oops, not sure why this got tagged to the query lines... this query worked for me after above fixes)